src/settings/Settings.tsx

```diff
@@ -645,6 +645,21 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
         default: "default",
     },
+    "webRtcAudio_autoGainControl": {
```

Comment:
Could you please use the same prefix as the other WebRTC settings (or `webrtc_audio_...`)?